picongpu_setup_KHI/include/picongpu/param/speciesInitialization.param

```diff
@@ -40,14 +39,14 @@ namespace picongpu
          * the functors are called in order (from first to last functor)
          */
         using InitPipeline = pmacc::mp_list<
-            CreateDensity<densityProfiles::Homogenous, startPosition::Random, PIC_Electrons>,
+            CreateDensity<densityProfiles::Homogenous, startPosition::Quiet, PIC_Electrons>,
             Derive<PIC_Electrons, PIC_Ions>,
-            Manipulate<manipulators::AssignXDriftPositive, PIC_Ions, filter::LowerQuarterYPosition>,
-            Manipulate<manipulators::AssignXDriftNegative, PIC_Ions, filter::MiddleHalfYPosition>,
-            Manipulate<manipulators::AssignXDriftPositive, PIC_Ions, filter::UpperQuarterYPosition>,
-            Manipulate<manipulators::AssignXDriftPositive, PIC_Electrons, filter::LowerQuarterYPosition>,
-            Manipulate<manipulators::AssignXDriftNegative, PIC_Electrons, filter::MiddleHalfYPosition>,
-            Manipulate<manipulators::AssignXDriftPositive, PIC_Electrons, filter::UpperQuarterYPosition>,
```

Comment:
@PrometheusPi Is this actually adding the small setup or replacing the big one? It looks like this changes the position of the shear planes also for the big setup...